code/go/0chain.net/smartcontract/dbs/goose/migrations/1700159742_remove_unused_cols_in_blobber.sql

```diff
@@ -6,6 +6,7 @@ ALTER TABLE blobbers DROP COLUMN IF EXISTS logo_url;
 ALTER TABLE blobbers DROP COLUMN IF EXISTS description;
 ALTER TABLE blobbers DROP COLUMN IF EXISTS latitude;
 ALTER TABLE blobbers DROP COLUMN IF EXISTS longitude;
+ALTER TABLE blobbers DROP COLUMN IF EXISTS used;
```

Comment:
Please make a new migration file for this change. Editing migration files may lead to ignoring your new changes if you tried to deploy your changes to a n/w without db cleanup.

Reply:
Done.